### DISCUSSION TITLE
Welcome to Financial-Analyzer Discussions!

### DISCUSSION THREAD

**Discussion by andersbandt** [andersbandt/Financial-Analyzer#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by edwrand** [comment]
Hi everyone, my name is Ed. I am a Python developer and am excited to help out. I think the front end UI can be improved so I will focus my effort there as I continue to read through the code base and learn more about the project!
